forward/forwarder_test.go

```diff
@@ -145,8 +255,7 @@ func (s *ForwarderTestSuite) TestKeysDiverged() {
 func (s *ForwarderTestSuite) TestRequestTimesOut() {
 	var ping Ping
 
-	s.sender.lookup = "127.0.0.2:3001"
-	dest, err := s.sender.Lookup("some key")
+	dest, err := s.sender.Lookup("unreachable")
```

Comment:
Yes, because this test is actually testing the behavior when the tcp connection is not being accepted. But if you look closely it is using an ip that is not normally in use, 127.0.0.**2**. I will test and see if we can use an ip from the test block that @dansimau recently send around.